cave/plot/plotter.py

```diff
@@ -189,6 +205,8 @@ def prepare_data(x_data):
             f.tight_layout()
             f.savefig(out)
             plt.close(f)
+        if len(output_fn) == 1:
```

Comment:
This seems odd to me.
If you only have one output_fn you return a string (the first element of a list).
But if the length is != 1 then you return a list. Why is that?
I'd try to always return the same type. This is always less error prone

Reply:
True. I left it as it was because I plan to rework scatter and cdf anyway. This was more of a hotfix to reenable the report with only train-instances.

Reply:
I see. For now it's ok but we shouldn't lose track of it, as it will make future maintenance more difficult